paper/paper.md

```diff
@@ -68,6 +68,10 @@ The package that comes the closest is Enzyme-JAX [@Moses2024].
 Enzyme-JAX allows one to differentiate a C++ function with Enzyme [@Moses2020; @Moses2021; @Moses2022] and connect it together with its derivative to JAX.
 However, it enforces the use of Enzyme for deriving derivatives and does not allow for connecting arbitrary code to JAX.
 
+PyTorch [@PyTorch2024] and TensorFlow [@tensorflow2015] also provide interfaces for custom extensions.
+PyTorch has an extensively documented Python interface to wrap custom Python functions as PyTorch functions.
+TensorFlow allows to expose a custom C++ function as a TensorFlow Op via a C++ interface.
```

Comment:
You might add a couple of sentences contrasting their approaches. How does `jaxbind` compare?

Reply:
Thanks for your comment! I expanded the description of the PyTorch and Tensorflow interfaces, clarifying how they compare to JAXbind.
As the paper is already on the long side for a JOSS publication, I would like to leave it at this level of detail. I added links to the documentation for readers interested in the details of the interfaces.